src/resources/js/docs.js

```diff
@@ -50,9 +50,6 @@ $(function() {
 		});
 
 	// Add links to [<matcher>] or named matcher tokens in code blocks.
-	// We can't do it exactly the same as the above block, because
-	// the matcher text includes <> character which are parsed as HTML
-	// unless we use text() to change the link text.
```

Comment:
Is this not true anymore?

Reply:
It was while I was writing it but then I changed things and force pushed in the last PR and it wasn't true anymore. Only noticed after it was merged.

Reply:
Really only the 2nd line of this comment was no longer valid (i.e. referring to "same as the above block") but the note about `text()` was still valid, I put that back in.